mkdocs.yml

```diff
@@ -57,6 +57,13 @@ pages:
   - Troubleshooting: extensions/troubleshooting.md # page-tree = DONE
   - Advanced patterns: extensions/advanced.md
 - Framework Reference:
```

Comment:
I propose changing this section to the following: 

```yaml
  - AngularJS:
    - AngularJS: framework/angular/index.md
    - AngularJS Quick Start: framework/angular/quickstart.md
    - AngularJS File Names: framework/angular/files.md
    - AngularJS Loader: framework/angular/loader.md
    - AngularJS Modifications:  framework/angular/modify.md

```

Reasons: 
* Use `index.md` for [hackable URLs](https://docs.civicrm.org/dev/en/master/documentation/style-guide/#hackable-urls) (This change also requires a file rename) 
* Use more descriptive menu item labels (as described [here](https://docs.civicrm.org/dev/en/master/documentation/style-guide/#parts-chapters-sections), so that the `<title>` element better reflects the page content).